odl/contrib/mrc/mrc.py

```diff
@@ -20,7 +20,7 @@
 import struct
 import warnings
 
-from odl.tomo.data.uncompr_bin import (
+from .uncompr_bin import (
```

Comment:
The recommendation is to always do `odl.contrib.mrc.uncompr_bin`, any reason you went absolute paths here?